src/content/platform-integration/web/wasm.md

```diff
@@ -207,9 +207,13 @@ static JS interop:
 - [`package:web`][], which replaces `dart:html` (and other web libraries)
 - [`dart:js_interop`][], which replaces `package:js` and `dart:js`
 
-Most packages owned by the Dart and Flutter teams have been
-migrated to be compatible with Wasm support in Flutter,
+The Dart and Flutter teams have migrated most packages
+to be compatible with Wasm support in Flutter,
 such as [`package:url_launcher`][].
+You can find packages that are compatible with Wasm
+by using the [`wasm-ready`][] filter on [pub.dev][].
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
To find Wasm-compatible packages, 
use the [`wasm-ready`][] filter on [pub.dev][].
```